semgrep/tests/e2e/rules/nosem.yaml

```diff
@@ -2,5 +2,10 @@ rules:
 - id: test-nosem
   message: test-nosem-message
   severity: WARNING
-  languages: [javascript]
-  pattern: nosem_func(...)
+  languages: [c, go, javascript, python]
+  pattern: test_nosem_func(...)
+- id: test-nosem-java
```

Comment:
This Java rule is currently split out because of https://github.com/returntocorp/semgrep/issues/1051